src/ResourceManagement/RecoveryServices/Microsoft.Azure.Management.RecoveryServices/project.json

```diff
@@ -0,0 +1,45 @@
+{
+  "version": "4.0.1-preview",
```

Comment:
@dragonfly91 if the current version is 3.0.1-preview, why is it going to 4.0.1-preview?